ocaml/xapi/repository_helpers.ml

```diff
@@ -628,6 +622,22 @@ module UpdateInfoMetaData = struct
     )
 end
 
+module UpdateInfoMetaData = struct
+  include RepoMetaData
```

Comment:
I don't think exposing all the functions in RepoMetadata is to our convenience here, I'd rather be explicit about the functions define in the module.